src/zenml/integrations/aws/__init__.py

```diff
@@ -20,10 +20,8 @@
 """
 from typing import List, Type
 
-from zenml.enums import StackComponentType
 from zenml.integrations.constants import AWS
 from zenml.integrations.integration import Integration
-from zenml.models import FlavorModel
```

Comment:
Why didn't the linter complain about these unused imports so far ?

Reply:
:magic_wand: